agent/metrics_test.go

```diff
@@ -32,6 +31,7 @@ func (s *MetricsTestSuite) SetupTest() {
 }
 
 func (s *MetricsTestSuite) TestRunForIntervalAndSendMessages() {
+	s.T().Skip("skipping on windows")
```

Comment:
This (and the ones below) should probably check if we're on windows before skipping

Reply:
Done